.github/workflows/build-deploy-staging.yaml

```diff
@@ -15,146 +15,109 @@ on:
     needs: [backend,harbor]
     strategy:
       matrix:
         target:
-          - {chain: main, network: main-stage}
-          - {chain: test, network: test-stage}
+        - {chain: main, network: main-stage}
+        - {chain: test, network: test-stage}
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Create ${{ matrix.target }} namespace
-        uses: mobilecoinofficial/gha-k8s-toolbox@v1
-        with:
-          action: namespace-create
-          namespace: reserve-auditor-staging
-          rancher_cluster: ${{ env.TARGET_CLUSTER }}
-          rancher_url: ${{ secrets.RANCHER_B_URL }}
-          rancher_token: ${{ secrets.RANCHER_B_TOKEN }}
-
-      - name: Deploys ${{ matrix.target.chain }}net release
-        uses: mobilecoinofficial/gha-k8s-toolbox@v1
-        with:
-          action: helm-deploy
-          rancher_cluster: ${{ env.TARGET_CLUSTER }}
-          namespace: reserve-auditor-staging
-          release_name: ${{ matrix.target.chain }}net-${{ env.REPO_NAME }}
-          rancher_url: ${{ secrets.RANCHER_B_URL }}
-          rancher_token: ${{ secrets.RANCHER_B_TOKEN }}
-          chart_repo: ${{ env.CHART_REPO }}
-          chart_name: ${{ env.REPO_NAME}}
-          chart_version: ${{ github.ref_name }}
-          chart_values: ${{ env.CHART_PATH }}/${{ matrix.target.chain }}net-values.yaml
-          chart_set: |
-            --set=mobilecoind.image.tag=${{ github.ref_name }}-${{ matrix.target.chain }}net
-            --set=frontend.image.tag=${{ github.ref_name }}
-            --set=ingress.host=auditor.stage.${{ matrix.target.chain }}.mobilecoin.com
-            --set=frontend.network=${{ matrix.target.network }}
+    - name: Checkout
+      uses: mobilecoinofficial/gh-actions/checkout@v0
+
+    - name: Create reserve-auditor-staging namespace
+      uses: mobilecoinofficial/gha-k8s-toolbox@v1
+      with:
+        action: namespace-create
+        namespace: reserve-auditor-staging
+        rancher_cluster: ${{ env.TARGET_CLUSTER }}
+        rancher_url: ${{ secrets.RANCHER_B_URL }}
+        rancher_token: ${{ secrets.RANCHER_B_TOKEN }}
+
+    - name: Deploys ${{ matrix.target.chain }}net release
+      uses: mobilecoinofficial/gha-k8s-toolbox@v1
+      with:
+        action: helm-deploy
+        rancher_cluster: ${{ env.TARGET_CLUSTER }}
+        namespace: reserve-auditor-staging
+        release_name: ${{ matrix.target.chain }}net-${{ env.REPO_NAME }}
+        rancher_url: ${{ secrets.RANCHER_B_URL }}
+        rancher_token: ${{ secrets.RANCHER_B_TOKEN }}
+        chart_repo: ${{ env.CHART_REPO }}
+        chart_name: ${{ env.REPO_NAME}}
+        chart_version: ${{ github.ref_name }}
+        chart_values: ${{ env.CHART_PATH }}/${{ matrix.target.chain }}net-values.yaml
+        # CBB: change mobilecoind.image to reserveAuditor.image...
+        chart_set: |
+          --set=mobilecoind.image.tag=${{ github.ref_name }}-${{ matrix.target.chain }}net
```

Comment:
is it ok that `--set=frontend.image.tag....` went away?

Reply:
I see it also went away in `dispatch-deploy.yaml`

Reply:
Should be. The `chart_version` sets the default for the frontend image tag, no need so specify it again.